pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/json/ImmutableJsonIndexReader.java

```diff
@@ -129,7 +135,8 @@ private MutableRoaringBitmap getMatchingFlattenedDocIds(FilterContext filter) {
       case AND: {
         List<FilterContext> children = filter.getChildren();
         int numChildren = children.size();
-        MutableRoaringBitmap matchingDocIds = getMatchingFlattenedDocIds(children.get(0));
+        MutableRoaringBitmap matchingDocIds =
+            getMatchingFlattenedDocIds(children.get(0));
```

Comment:
(format) Please reformat all the changes

Reply:
Ack